src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.html

```diff
@@ -69,15 +69,17 @@ <h5>{{ chartTime }}</h5>
                 <div class="btn-group">
                     <div
                         class="btn"
-                        [ngClass]="{ 'btn-primary selected': !showLifetimeOverview, 'btn-default': showLifetimeOverview }"
+                        [class.btn-primary.selected]="!showLifetimeOverview"
+                        [class.btn-default]="showLifetimeOverview"
```

Comment:
This should also work and is cleaner in my opinion
[ngClass]="{ showLifetimeOverview ? 'btn-primary selected' : 'btn-default' }"

Reply:
I assume this is personal preference. 😄

Reply:
But, this is clearly an improvement in file length efficiency. Changed in b202aa1.